ftw/referencewidget/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<registry>
+    <records interface="ftw.referencewidget.interfaces.IReferenceSettings" />
+
+    <records prefix="plone.resources/referencewidget_css" interface="Products.CMFPlone.interfaces.IResourceRegistry">
+        <value key="css">
+            <element>++resource++ftw.referencewidget/refwidget.css</element>
+        </value>
+    </records>
+
+    <records prefix="plone.resources/handlebar_js" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="js">++resource++ftw.referencewidget/handlebars.js</value>
+        <value key="deps">jquery</value>
+    </records>
+
+    <records prefix="plone.resources/refwidget_js" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="js">++resource++ftw.referencewidget/refwidget.js</value>
+        <value key="deps">jquery</value>
+    </records>
+
+
+    <records prefix="plone.bundles/ftw-referencewidget-resources" interface="Products.CMFPlone.interfaces.IBundleRegistry">
+        <value key="resources">
+            <element>handlebar_js</element>
+            <element>refwidget_js</element>
+            <element>referencewidget_css</element>
+        </value>
+        <value key="enabled">True</value>
+        <value key="depends">plone-legacy</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++plone++referencewidget/ftw-referencewidget-compiled.js</value>
```

Comment:
```suggestion
```
The `jscompilation` option is for when we want to do the compilation outside of Plone and actually store this file on the filesystem.

See https://docs.plone.org/adapt-and-extend/theming/resourceregistry.html#compiling-bundles

> **Use your own compilation chain**
> 
> The Plone 5 Resource Registry can be used with your favorite build system. Use the tool you prefer create a compiled version of your bundle. Your bundle registration must provide a URL for the “jscompilation” and “csscompilation” options. Your compiled files must be in the filesystem locations that are indicated by these values.

So we should not use this option since we are not compiling the resource.

Reply:
@jone are you sure on that? I guessed that we would be using plone-compile-resources (or our own toolchain) - about which the docs say:

> If you use the plone-compile-resources script, or your own custom toolchain to compile your own bundle JS or CSS, you will need to manage these values yourself.

> **jscompilation**
> ...